aquality-automation/aquality-tracking-ui
# Feature/suite matrix does not work

# PR Details

Fix for Suite Matrix

## Related Issue

#18 

## How Has This Been Tested

Precondition: There is project with multiple testruns assigned to one suite

1. Open suite matrix
1. Select Suite from precondition
1. Click Show

Expected Result: Table with results for test runs from suite is shown

## Checklist
